documentation/contributing/git-tips.adoc

```diff
@@ -103,3 +103,20 @@ $ git rebase <user>/<merge-request-branch>
 First, rewinding head to replay your work on top of it...
 Fast-forwarded <new-topic-branch> to <user>/<merge-request-branch>
 ----
+
+=== Committing to another writer's branch
+
+If you have made changes to the content in someone else's branch you'll want to be sure to commit them back to that same branch/PR.
+
+. While in your working branch, commit your changes
++
+[source,options="nowrap",subs="+quotes"]
+----
+$ git commit -m "__MESSAGE__"
+----
+. Push your changes to the existing PR
++
+[source,options="nowrap",subs="+quotes"]
+----
+$ git push _USER_ HEAD:__MERGE_REQUEST_BRANCH__
```

Comment:
I think this will not work in 99% cases because you are not allowed to push into someone else's branch, or?

Reply:
Is the command maybe wrong? Because you're actually pushing to your copy of the other person's branch.  I believe I've followed these instructions before.

Reply:
Well, AFAIK that would work only when it is in the main repo. But not when it would be in fork. Because that would be a security issue.

Reply:
@tombentley and @laidan6000 what steps were you following when working together in the same branch?

Reply:
@ncbaratta That was just a branch in the same repo. That was not a fork. This is something we can do between us. But not something generally applicable.

Reply:
What would work would be for person A to push to their fork, and then for B to fetch A's branch, having added A's fork as a remote) and then open a PR against A's fork. That's not what is described here, and would be a more onerous workflow for people who have push access to a common repo.

Reply:
Wait ... that's what the instructions say.  This section comes right after : https://strimzi.io/contributing/guide/#access_another_writers_unmerged_commits

Reply:
Yes, but it describes just a checkout. Not the changes etc.